nautobot/extras/forms.py

```diff
@@ -623,6 +756,14 @@ class NautobotModelForm(BootstrapMixin, CustomFieldModelForm, RelationshipModelF
     """
 
 
+class NautobotFilterform(BootstrapMixin, CustomFieldFilterForm, RelationshipModelFilterForm):
```

Comment:
```suggestion
class NautobotFilterForm(BootstrapMixin, CustomFieldFilterForm, RelationshipModelFilterForm):
```

`nit`

Also, why the Mixin first? Is it the proper base model?

Be sure to update where this is referenced everywhere.

Reply:
`BootstrapMixin` usually goes first in any of the forms and it is inherited from `form.BaseForm `so I think it should be the proper base model